apps/emqx_authz/src/emqx_authz_api_sources.erl

```diff
@@ -154,7 +154,15 @@ source_api() ->
                     name => type,
                     in => path,
                     schema => #{
-                       type => string
+                       type => string,
+                        enum => [ <<"file">>
```

Comment:
make a function for this instead of copy-paste.

Reply:
I don't have any good ideas, can you hint at them ?

Reply:
resolve this issue for now. should fix in future PRs when use hocon schema for http API